doc/source/api-reference/qibo.rst

```diff
@@ -2414,51 +2414,155 @@ Alternatively, a Clifford circuit can also be executed starting from the :class:
     :members:
     :member-order: bysource
 
-Gate set tomography
-"""""""""""""""""""
-
-Gate set tomography is used to obtain the noise profile of each operator.
-The noise profile can then be inverted and encoded in basis operations to be
-used in probabilistic error cancellation.
+.. _Gate Set Tomography:
 
-.. testsetup:: python
+Gate Set Tomography
+-------------------
 
-    from qibo.noise import NoiseModel, DepolarizingError
-    from qibo import Circuit
-    from qibo import gates
-    from qibo.tomography.gate_set_tomography import *
-    import numpy as np
+The gate set tomography (GST) follows the procedure outlined in
+https://arxiv.org/pdf/1712.09271.pdf, which follows closely to that of the
+Linear inversion GST given in https://arxiv.org/pdf/1509.02921.pdf. The outputs
+of the functions, when used correctly, will allow one to set up the proper
+variables necessary for probabilistic error cancellation to mitigate noise
+arising from noisy quantum hardware. The codes allow for calibrating an empty
+circuit, and for one and two qubits of gate set tomography. We give some
+examples to illustrate the usage of the functions for clarity.
+
+The ``reset_registers()`` function in
+:class:`qibo.tomography.gate_set_tomography` requires a string argument serving
+as the identifier for the qubit number(s) from which the inverse circuit will
+be obtained. This identifier ensures that the function returns an inverse
+circuit of the specified register, resulting in the preparation of the zero
+state :math:`| 0 \rangle`. It's important to note that this function is
+specifically designed for qubit(s) without entangling gates."
```

Comment:
May I propose that we remove "reset_registers()" and include it in the next PR for PEC? I think "reset_registers()" is used in the GST of the basis operations which is part of PEC.

Reply:
Actually let's keep "reset_registers()" there as part of state preparation. I will change the input for "reset_registers()" to tuples instead of strings.